src/main/java/org/zalando/nakadi/service/subscription/zk/NewZkSubscriptionClient.java

```diff
@@ -155,18 +155,25 @@ public void updatePartitionsConfiguration(final Partition[] partitions) throws N
     }
 
     private Topology readTopology() throws Exception {
```

Comment:
No runtime exceptions here. Also I think it is better to process Exception in this method and throw runtime exception as well, because it is what you basically do in the methods which use this method.